stable/autonity-network/README.md

```diff
@@ -2,130 +2,123 @@
 
 [![Join the chat at https://gitter.im/clearmatics/autonity](https://badges.gitter.im/clearmatics/autonity.svg)](https://gitter.im/clearmatics/autonity?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-
 ## Introduction
+This chart deploys a **private** [Autonity](https://www.autonity.io/) network onto a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+"Ethereum based protocol enabling permissioned, decentralized and interoperable transacting member-mutual networks." - [Autonity](https://www.autonity.io)
+
+## Examples
+### Prerequisites
+1. Install [Kubernetes](http://kubernetes.io) `v1.18.4`. Basic features have been tested in both Minikube and Kind with extra functionality available in EKS and GKE:
+   - Local: [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+   - Local: [Kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+   - Cloud: [Amazon EKS](https://eksworkshop.com/prerequisites/self_paced/)
+   - Cloud: [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+1. Install Helm [3.2.4](https://github.com/helm/helm/releases/tag/v3.2.4)
+1. Initialise [the official](https://helm.sh/docs/intro/quickstart/ the official) `@stable` and the Autonity Helm charts repositories:
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
Why are we need it?

Reply:
[No repositories are shipped with Helm 3.](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/)